Platforms/QemuQ35Pkg/ConfigKnobs/ConfigDataGfx/ConfigDataGfx.c

```diff
@@ -64,16 +64,23 @@ ApplyGfxConfigToPolicy (
   GfxEnablePort0   = *(BOOLEAN *)ConfigBuffer;
   GfxSiliconPolicy = AllocateCopyPool (sizeof (DefaultQemuGfxPolicy), DefaultQemuGfxPolicy);
 
+  if (GfxSiliconPolicy == NULL) {
+    DEBUG ((DEBUG_ERROR, "Failed to allocate Policy structure\n"));
+    goto Exit;

```

Comment:
It looks like `Status` will be uninitialized in this `goto Exit` path.

```suggestion
  if (GfxSiliconPolicy == NULL) {
    DEBUG ((DEBUG_ERROR, "Failed to allocate Policy structure\n"));
    Status = EFI_OUT_OF_RESOURCES;
    goto Exit;
```

Should add `EFI_OUT_OF_RESOURCES` to the function return values if this is done.

Reply:
Good suggestion. Updated w/ recommendations.